File: lib/plateau.rb
class Plateau
  attr_accessor :coords_bottom_left, :coords_top_right
  def initialize(b_left, t_right)
    @coords_bottom_left = b_left.split(',').map(&:to_i)
    @coords_top_right   = t_right.split(',').map(&:to_i)
  end
  def size
    [@coords_bottom_left, @coords_top_right]
  end
end

File: Marsrovers/lib/src/net/torchz/mars/Plateau.java
package net.torchz.mars;

import java.util.Arrays;

/**
 * Created with IntelliJ IDEA.
 * User: tcz
 * Date: 12-12-19
 * Time: 上午10:25
 * To change this template use File | Settings | File Templates.
 */
public class Plateau {
    private Coords coords_bottom_left;
    private Coords coords_top_right;
    public Plateau(String bottom_left, String top_right){
        coords_bottom_left = new Coords(bottom_left.split(","));
        coords_top_right = new Coords(top_right.split(","));
    }

    public Coords getCoords_bottom_left() {
        return coords_bottom_left;
    }

    public Coords getCoords_top_right() {
        return coords_top_right;
    }
}
